src/rprblender/utils/helper_lib.py

```diff
@@ -60,7 +61,15 @@ def init():
 
     lib_path = next(p / lib_name for p in paths if (p / lib_name).is_file())
     log('Load lib', lib_path)
-    lib = ctypes.cdll.LoadLibrary(str(lib_path))
+    try:
+        lib = ctypes.cdll.LoadLibrary(str(lib_path))
+    except OSError as e:  # expand the traceback info with the exact addon version and library name
+        raise Exception(f"Failed to load library {lib_path}",
```

Comment:
What do you think of just logging this error and provide plugin to be in working state? Of course openvdb and sun&sky functionality won't be working, but everything else will be working. 
@bsavery what do you think about it?

Reply:
I mean, it depends on the library.  If RPR lib fails that's bad.  But hard to tell if other libraries don't fail what is going to work and what won't

Reply:
So a "borked" load is hard to predict.  and might be ok just to fail